### PR TITLE
Added 'Strings added in Underscore templates' section to 'Translate theme strings' page

### PR DESCRIPTION
**Pull request by sergiy-v**
## Purpose of this pull request

This pull request (PR) adds the 'Strings added in Underscore templates' section to the 'Translate theme strings' page according to the following [PR](https://github.com/magento/magento2/pull/26435).

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/translate_theory.html

## Links to Magento source code
https://github.com/magento/magento2/blob/2.4-develop/lib/web/mage/translate.js#L50
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Translation/etc/di.xml#L67

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md [hunk 1]
```diff
@@ -100,6 +100,24 @@ Translated strings that originate from `.xml` files will not render unless they
...
+
+To ensure that the text you add in `.html` Underscore templates is collected by the `i18n` tool and added to the dictionary, use the `_.i18n('')` Underscore function.
+
+-  When only a string is added in template use the following example:
+
+   ```html
+   <%= _.i18n('Hello') %>
+   ```
+
+-  When a string contains a variable, to add a placeholder for this variable to the string stored in the dictionary, use the syntax similar to the following:
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md]: Please, simplify this statement

```suggestion
-  When the string contains a variable, use the variable placeholder for processing string stored in the dictionary: 
```

Comment (sergiy-v) [reply src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md]: Thank you for suggestion, the changes has been added.

**Comment by dobooth** [comment]
running tests

**Pull request merged**